File: isegoria/src/main/java/com/eulersbridge/isegoria/models/VoteLocation.java
package com.eulersbridge.isegoria.models;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

@SuppressWarnings("unused")
@Parcel
public class VoteLocation {

    public long ownerId;

    @Json(name = "votingLocationId")
    public long id;

    public String name;
    public String information;

    @Override
    public String toString() {
        return name;
    }
}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/VoteLocation.kt
package com.eulersbridge.isegoria.network.api.models

import android.annotation.SuppressLint
import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class VoteLocation (
    val ownerId: Long = 0,
    @field:Json(name = "votingLocationId") var id: Long = 0,
    val name: String?,
    val information: String?

) : Parcelable {
    override fun toString()
        = name ?: id.toString()
}